### ISSUE TITLE
New Gas Clouds obstacles missing

### ISSUE THREAD

**Issue by techillin** [geordanr/xwing-miniatures-font#104]
https://raithos.github.io/images/gascloud4.png
https://raithos.github.io/images/gascloud5.png
https://raithos.github.io/images/gascloud6.png

**Comment by ScottKarch** [comment]
It looks like they are in there
![image](https://user-images.githubusercontent.com/19825797/73104710-adddf600-3ebc-11ea-9a69-ccd5893b76a5.png)

**Comment by techillin** [comment]
Those are the old ones. Three new ones came in the new obstacles pack for a total of six.

**Comment by ScottKarch** [comment]
Wow...  I apologize, I didn't look closely at the images you posted.  I'll see if I can use those images.  I appreciate it

**Comment by ScottKarch** [comment]
![image](https://user-images.githubusercontent.com/19825797/73116875-52832680-3f03-11ea-93ff-0eb7cdb945eb.png)

Gas Cloud 4 Outline
U+0115
ĕ

Gas Cloud 5 Outline
U+0116
Ė

Gas Cloud 6 outline
U+0117
ė

Gas Cloud 4
U+0131
ı

Gas Cloud 5 
U+0132
Ĳ

Gas Cloud 6
U+0133
ĳ

[xwing-miniatures 2020 Jan 24.ttf.zip](https://github.com/geordanr/xwing-miniatures-font/files/4111657/xwing-miniatures.2020.Jan.24.ttf.zip)

**Comment by techillin** [comment]
Looks great except 4 and 5 switched places (if it matters).

**Comment by ScottKarch** [comment]
Yup..   thanks. I'll tweak it tonight or tomorrow.   Thanks for noticing.

**Comment by techillin** [comment]
These three symbols have not yet been pushed to the project. Anyone can help with that?

**Comment by geordanr** [comment]
These glyphs will need to be imported into the current version of the TTF (which had the negative recurring icon added in the meantime).  I don't know how to do manipulate TTFs, but if someone can do this and provide the merged TTF, I can update the CSS to match.
